src/pages/StatusPage.vue

```diff
@@ -77,6 +77,16 @@
                     <font-awesome-icon icon="save" />
                     {{ $t("Switch to Dark Theme") }}
                 </button>
+
+                <button v-if="tagsVisible == 'hidden'" class="btn btn-secondary me-2" @click="changeTagsVisibilty('visible')">
```

Comment:
So, if someone is using tabbing to navigate the UI, having 2 separate buttons for the options may cause some strange behavior. Can you combine these into one button that determines what is shown/done based on the current state? 

Also, if possible, please add a tool tip that says "currently displaying tags/not displaying tags". Different UIs treat what the icon represents differently. IIRC Most use it as "make it go to this state" but some others do "it is currently in this state". The button text should cover it, but having a tool tip basically eliminates the ambiguity all together.

Reply:
> having 2 separate buttons for the options may cause some strange behaviour

I also thought that at first glance, but it doesn't doesn't because `v-if` doesn't hide the element, it completely removes it from the DOM. `v-show` might cause this behaviour.

But yeah, I agree with combining the buttons, I was just trying to maintain consistency with the existing code-base, and the theme button is laid out like that. I wonder if I should also refactor the theme button to the same style while I'm at it?

> add a tool tip

Yup, great idea, but none of the other buttons have tool tips, might be a good idea to make that a separate PR?

> IIRC Most use it as "make it go to this state"

Yeah, the "make it go to this state" is much more common and accepted, but it is always good to clear ambiguity.

Here is the new combined button (tagsVisible is going to be a bool now):
```html
<button class="btn btn-secondary me-2" @click="changeTagsVisibilty(!tagsVisible)">
	<template v-if="tagsVisible">
		<font-awesome-icon icon="eye" />
		{{ $t("Hide Tags") }}
	</template>
	<template v-else>
		<font-awesome-icon icon="eye-slash" />
		{{ $t("Show Tags") }}
	</template>
</button>
```

Reply:
I think it getting removed from the DOM would throw off the tab highlighting as there was nothing left _to_ highlight, which was the problem. Its fixed now regardless.

As for the tool-tips it's up to @louislam, but if he is ok with us having them, IMO we should add them right away, and then whenever touching old code (mind, only when reasonable, like with adding tests) instead of as one big PR.